src/ifElse.test.js

```diff
@@ -1,11 +1,42 @@
 'use strict';
 
 describe('ifElse', () => {
-  // const { ifElse } = require('./ifElse');
+  const { ifElse } = require('./ifElse');
+  let condition, first, second;
 
-  it('should ', () => {
+  beforeEach(() => {
+    condition = jest.fn(() => true);
+    first = jest.fn();
+    second = jest.fn();
+  });
+
+  it('should take 3 callbacks ', () => {
+    expect(typeof condition).toBe('function');
+    expect(typeof first).toBe('function');
+    expect(typeof second).toBe('function');
+  });
+
+  it('should call second if condition is true', () => {
+    ifElse(condition, first, second);
+
+    expect(condition).toHaveBeenCalled();
+    expect(first).toHaveBeenCalled();
+    expect(second).not.toHaveBeenCalled();
```

Comment:
There is a logical error in this test case description. The test case is named 'should call second if condition is true', but it actually tests that 'first' is called when the condition is true. You might want to update the test case description to reflect the correct behavior.